spec/ar2dto/has_dto_spec.rb

```diff
@@ -24,13 +24,15 @@
   describe "#to_dto" do
     let(:attributes) do
       {
-        name: "Sandy",
```

Comment:
not for now but I wonder if we should split the test file.

Again, not for now 👍